### PR TITLE
Patch file to fix races in upstream gosigma library

### PR DESCRIPTION
**Pull request by wallyworld**
## Description of change

Various races attributed to the github.com/altoros/gosigma repo appear in Juju's CI test runs. The fixes are proposed upstream at https://github.com/Altoros/gosigma/pull/1
Until that code lands, this PR delivers a patch that is applied to Juju to fix the issues.

## QA steps

go test -race on altoros/gosigma repo

### CODE REVIEW

File: patches/README.md [hunk 1]
```diff
@@ -9,5 +9,5 @@ we need to apply sooner.
 They're applied with `$GOPATH/src` as the current directory, and with
 `-p1` to strip one component off the start of the file path.
 
-For more details, see `lp:juju-release-tools/apply_patches.py` or ask
-babbageclunk or mgz in IRC.
+For more details, see `https://github.com/juju/juju/releasetests/apply_patches.py`
+or ask babbageclunk in IRC.
```
Comment (babbageclunk) [inline patches/README.md]: Good call, Martin probably isn't super interested in hearing about this. I think balloons is an about-this-knower as well, may as well put him in here.

Comment (wallyworld) [inline patches/README.md]: done

**Comment by wallyworld** [comment]
$$merge$$

**Comment by wallyworld** [comment]
$$merge$$

**Comment by wallyworld** [comment]
$$Merge$$

**Pull request merged**